pages/auth/setup-security-questions.tsx

```diff
@@ -144,30 +138,30 @@ const SetupSecurityQuestions = ({ questions = [] }: { questions: Question[] }) =
                 <Label id={"label-question1"} htmlFor={"question1"} className="required" required>
                   {t("question")} 1
                 </Label>
-                <Select id="question1" name="question1" className="mb-0 w-full rounded">
-                  {
+                <Dropdown id="question1" name="question1" className="mb-0 w-full rounded">
+                  <>
                     <option key={"default"} value="">
                       {t("questionPlaceholder")}
                     </option>
-                  }
-                  {questions
-                    .filter((q) => {
-                      if (formRef?.current && formRef.current?.values) {
-                        if (
-                          q.id === formRef.current.values.question2 ||
-                          q.id === formRef.current.values.question3
-                        ) {
-                          return false;
+                    {questions
```

Comment:
Could destructure the incoming props here:

```
{questions
                      .filter(({ id }) => {
                        if (formRef?.current && formRef.current?.values) {
                          if (
                            id === formRef.current.values.question2 ||
                            id === formRef.current.values.question3
                          ) {
                            return false;
                          }
                        }
                        return true;
                      })
                      .map(({ id, question }) => (
                        <option key={id} value={id}>
                          {question}
                        </option>
                      ))}
```

Reply:
Good find - missed these. Thanks!